script0.js

```diff
@@ -0,0 +1,154 @@
+  } else {
+    makeTens = num2 * 10
+    console.log(`The number2 value in tens is,`, makeTens)
+    makeNum = makeTens + num1
+  }
+  return makeNum
+}
+
+mode = "roll dice"
+let dice1_1 = 0
+let dice1_2 = 0
+let dice2_1 = 0
+let dice2_2 = 0
+let playersNumbers = []
+console.log("aaaaaa")
+
+let play2DiceTogether = function(input) {
+  console.log("bbbbb")
+  console.log('start with', mode)
+    if (mode === "roll dice"){
+    dice1_1 = rollDice()
+    dice1_2 = rollDice()
+    console.log(`mode is`, mode)
+    mode = "p1 order dice"
+    console.log(`###############`)
+    return `Welcome ` +  (orderDiceMsg(1,dice1_1,dice1_2))
+  }
+  console.log("out of the roll dice mode")
+  console.log(`mode is`, mode)
+  console.log(`input`, input)
+  console.log("$$$$$$$$$$$$$$$$$")
+  if (mode === "p1 order dice" && input === '1'){
+    mode = "make p1 dice 1 big"
+    console.log(`p1 input`, input)
+    console.log(`mode is `, mode)
+    let makeNum = compoundNum(input, dice1_1,dice1_2)
+    // makeTens = dice1_1 * 10
+    // console.log(`The dice1 value in tens is,`, makeTens)
+    // makeNum = makeTens + dice1_2
+    playersNumbers.push(makeNum)
+    console.log(`Submitted score for plyaer 1 to compare`, playersNumbers)
+    mode = "player 2 turn"
+    console.log(mode,`@@@@@@@@@@@@@`)
+    return createTotalMsg(1,1,makeNum) + `It is now Player 2's turn.` 
+  } 
+  if (mode === "p1 order dice" && input === '2'){
+      mode = "make p1 dice 2 big"
+      console.log(`p1 input`, input)
+      console.log(`mode is `, mode)
+      let makeNum = compoundNum(input, dice1_1,dice1_2)
+      // makeTens = dice1_2 * 10
+      // console.log(`The dice2 value in tens is,`, makeTens)
+      // makeNum = makeTens + dice1_1
+      playersNumbers.push(makeNum)
+      console.log(`Submitted score for plyaer 1 to compare`, playersNumbers)
+      mode = "player 2 turn"
+      console.log(mode,`%%%%%%%%%%%%%`)
+      return createTotalMsg(1,2,makeNum) + `It is now Player 2's turn.` 
+    }
+  
+  if (mode === "player 2 turn"){
+    dice2_1 = rollDice()
+    dice2_2 = rollDice()
+    mode = "player 2 decide"
+    return  orderDiceMsg(2,dice2_1,dice2_2)
+  }
+  if (mode === "player 2 decide" && input === '1') {
+    mode = "make p2 dice 1 big"
+    console.log(`p2 input`, input)
+    console.log(`mode is `, mode)
+    let makeNum = compoundNum(input, dice2_1,dice2_2)
+    // makeTens = dice2_1 * 10
+    // console.log(`The dice1 value in tens is,`, makeTens)
+    // makeNum = makeTens + dice2_2
+    playersNumbers.push(makeNum)
+    mode = "results"
+    return createTotalMsg(2,1,makeNum)
+  }
+  if (mode === "player 2 decide" && input === '2') {
+    mode = "make p2 dice 2 big"
+    console.log(`p2 input`, input)
+    let makeNum = compoundNum(input, dice2_1,dice2_2)
+    // makeTens = dice2_2 * 10
+    // console.log(`The dice2 value in tens is,`, makeTens)
+    // makeNum = makeTens + dice2_1
+    playersNumbers.push(makeNum)
+    mode = "results"
```

Comment:
this parts are the similar to the code you have in from line 90 to line 96. can be written once, and have your conditional logic on line 86 and 98 apply for the 
return createTotalmsg on line 107